apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/TimestampBound.cs

```diff
@@ -224,22 +233,32 @@ public static TimestampBound FromBase64String(string base64String)
             {
                 throw new InvalidOperationException($"Unable to properly deserialize {nameof(TimestampBound)}.{nameof(Mode)}");
             }
+            if (structValue.Fields.TryGetValue(nameof(ReturnReadTimestamp), out Value returnReadTimestampValue))
+            {
+                returnReadTimestamp = Boolean.Parse(returnReadTimestampValue.StringValue);
+            }
+            TimestampBound timestampBound;
```

Comment:
All of this will be simplified back to return statements if we make this immutable again.